reconcile/queries.py

```diff
@@ -2132,3 +2132,68 @@ def get_pagerduty_instances():
 def get_gabi_instances():
     gqlapi = gql.get_api()
     return gqlapi.query(GABI_INSTANCES_QUERY)['gabi_instances']
+
+
+PERMISSIONS_QUERY = """
+{
+  permissions: permissions_v1 {
+    service
+    ...on PermissionSlackUsergroup_v1 {
+      channels
+      description
+      handle
+      ownersFromRepos
+      pagerduty {
+          name
+          instance {
+            name
+          }
+          scheduleID
+          escalationPolicyID
+        }
+      roles {
+        users {
+            name
+            org_username
+            slack_username
+            pagerduty_username
+        }
+    }
+      schedule {
+          schedule {
+            start
+            end
+            users {
+              org_username
+              slack_username
+            }
+          }
+        }
+      workspace {
+        name
+        token {
+          path
+          field
+        }
+        api_client {
+          global {
+            max_retries
+            timeout
+          }
+          methods {
+            name
+            args
+          }
+        }
+        managedUsergroups
+      }
+    }
+  }
+}
+"""
+
+
+def get_permissions():
+    gqlapi = gql.get_api()
+    permissions = gqlapi.query(PERMISSIONS_QUERY)['permissions']
+    return [p for p in permissions if p['service'] == 'slack-usergroup']
```

Comment:
Since the func name is get_permissions. I suggest we return all permissions here and filter slack-usergroup permissions inside the integration.

Reply:
Well the query itself is tailored for getting those information that's related to slack_usergroup integration, i.e. toke, api_client, especially it is querying `PermissionSlackUsergroup_v1`.  There could be a use case that need to query permission with more information. How do you feel about renaming this function to get_permissions_for_slack_usergroup?

Reply:
get_permissions_for_slack_usergroup works for me